openstack/networking/v2/extensions/fwaas_v2/rules/requests.go

```diff
@@ -90,6 +159,47 @@ func Get(c *gophercloud.ServiceClient, id string) (r GetResult) {
 	return
 }
 
+// UpdateOptsBuilder is the interface options structs have to satisfy in order
```

Comment:
I think this PR might need rebased off of Gophercloud's master branch. Update has already been merged: https://github.com/gophercloud/gophercloud/blob/master/openstack/networking/v2/extensions/fwaas_v2/rules/requests.go#L93

Reply:
Whoops :D Good catch, thanks!